test/plain-list/AutosuggestApp.test.js

```diff
@@ -131,6 +132,83 @@ describe('Plain list Autosuggest', () => {
     });
   });
 
+  describe('when typing and matches exist (when selectFirstSuggestion is true)', () => {
+    before(() => {
+      selectFirstSuggestion.true();
+    });
+
+    after(() => {
+      selectFirstSuggestion.false();
```

Comment:
Updated in https://github.com/moroshko/react-autosuggest/pull/165/commits/ab3ba610685b26793f328af13e0818ebe0a4b148